signup.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="ko">
+    <head>
+        <title>판다 마켓</title>
+        <link rel="stylesheet" href="common.css"/>
+        <link rel="stylesheet" href="login.css"/>
+    </head>
+
+    <body>
+        <div class="container">
+            <a href="/"><img id="panda-market" src="/images/logo.png"/></a>
+
+            <div class="inner-container">
+                <div class="input-title">이메일</div>
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

```suggestion
                <label class="input-title" htmlFor="연결하고 싶은 인풋의 아이디">이메일</label>
```